models/demos/wormhole/stable_diffusion/tt2/ttnn_functional_cross_attention.py

```diff
@@ -737,14 +737,17 @@ def __call__(
             )
             # TODO: Output sharded once https://github.com/tenstorrent/tt-metal/issues/6775 is fixed
             interleaved_out = hidden_states.shape[-2] == 8192 or hidden_states.shape[-2] == 2048
-            qkv_out = ttnn.experimental.operations.primary.matmul(
+            hidden_states = ttnn.experimental.tensor.sharded_to_interleaved(
```

Comment:
The input to this matmul should not be interleaved.

Reply:
Fixed.